application/modules/invoices/controllers/Invoices.php

```diff
@@ -54,6 +54,12 @@ public function status($status = 'all', $page = 0)
             case 'paid':
                 $this->mdl_invoices->is_paid();
                 break;
+            case 'unpaid':
```

Comment:
should `archived` be added here as well?

Reply:
done